panther

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env php
+<?php
+
+/*
+ * This file is part of the Panther project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Psy\Shell;
+use Symfony\Component\Panther\Client;
+use Symfony\Component\Panther\PantherTestCaseTrait;
+
+require file_exists($autoload = __DIR__.'/vendor/autoload.php') ? $autoload : __DIR__.'/../../autoload.php';
+
+if (!class_exists(Shell::class)) {
+    $psyshBin = $_SERVER['PANTHER_PSYSH_BIN'] ?? strtok(exec(sprintf('%s psysh', '\\' === \DIRECTORY_SEPARATOR ? 'where' : 'command -v')), PHP_EOL);
+    if (file_exists($psyshBin)) {
+        require $psyshBin;
+        if (version_compare(Shell::VERSION, '0.9.9', '<=')) {
+            fwrite(STDERR, sprintf('Please upgrade the version of your PsySH script at "%s".', $psyshBin).PHP_EOL);
```

Comment:
What about this:

```suggestion
            @trigger_error(sprintf('Please upgrade the version of your PsySH script at "%s" to version 0.9.9 or superior.', $psyshBin), E_USER_DEPRECATED);
```

It will allow the phpunit bridge and monitoring systems to catch the problem.

Reply:
Could you change this if appropriate (or resolve the discussion if not @maidmaid)? I would like to merge this one and tag a new release before SymfonyCon.

Reply:
I just did it :)